projects/ad9083_vna/common/ad9083_vna_bd.tcl

```diff
@@ -0,0 +1,92 @@
+
+# RX parameters
+set RX_NUM_OF_LANES 1        ; # L
+set RX_NUM_OF_CONVERTERS 32  ; # M
+set RX_SAMPLES_PER_FRAME 1   ; # S
+set RX_SAMPLE_WIDTH 16       ; # N/NP
```

Comment:
Since most of the JESD204 based projects have been updated, the RX parameters should be exposed in system_top.tcl so they can be changed at make time. 
See references: https://github.com/analogdevicesinc/hdl/blob/master/projects/daq2/zcu102/system_project.tcl
The same changes were done in this PR too: https://github.com/analogdevicesinc/hdl/pull/914

Reply:
Because the change will affect ad9083 as well. I prefer to do the changes for both after merging this PR in a separate PR.